src/it/java/org/jboss/tools/intellij/quarkus/utils/QuarkusUtils.java

```diff
@@ -76,4 +79,30 @@ public static void selectQuarkusExtensions(NewProjectDialogFixture newProjectDia
             }
         });
     }
+
+    public static String createNewJavaProjectWithQuarkusFramework(RemoteRobot remoteRobot, String projectName) {
+        final WelcomeFrameDialogFixture welcomeFrameDialogFixture = remoteRobot.find(WelcomeFrameDialogFixture.class);
+        welcomeFrameDialogFixture.createNewProjectLink().click();
+        final NewProjectDialogFixture newProjectDialogFixture = welcomeFrameDialogFixture.find(NewProjectDialogFixture.class, Duration.ofSeconds(20));
+        NewProjectDialogUtils.selectNewProjectType(remoteRobot, "Java");
+
+        newProjectDialogFixture.theFrameworksTree().findText("Quarkus").click();
+        Point quarkusCheckboxLocation = newProjectDialogFixture.theFrameworksTree().findText("Quarkus").getPoint();
+        quarkusCheckboxLocation.x = 10;
+        newProjectDialogFixture.theFrameworksTree().click(quarkusCheckboxLocation);
```

Comment:
Is there no way to actually find the checkbox ?

Reply:
There is no way to find the checkbox easily right now. The checkboxes are not accessible using the Remote-Robot, they are wrapped inside "ExtensionsTable" element. I suggest to leave it for now as it is. See:

<img width="748" alt="Snímek obrazovky 2021-02-02 v 23 47 57" src="https://user-images.githubusercontent.com/43820055/106673167-80d52080-65b1-11eb-8764-302b8c37c6e0.png">

<img width="1300" alt="Snímek obrazovky 2021-02-02 v 23 55 32" src="https://user-images.githubusercontent.com/43820055/106673651-31432480-65b2-11eb-8b00-cc2e68d46062.png">